addons/web/static/src/js/chrome/search_view.js

```diff
@@ -625,6 +610,34 @@ var SearchView = Widget.extend({
         fv.arch = utils.xml_to_json(doc, true);
         return fv;
     },
+    _setSearchViewButtons: function() {
+        var menu_defs = [];
+        this.prepare_search_inputs();
+        var $buttons = this._getButtonsElement();
+        if ($buttons) {
+            if (!this.options.disable_filters) {
+                this.filter_menu = new FilterMenu(this, this.filters, this.fields);
+                menu_defs.push(this.filter_menu.appendTo($buttons));
+            }
+            if (!this.options.disable_groupby) {
+                this.groupby_menu = new GroupByMenu(this, this.groupbys, this.fields);
+                menu_defs.push(this.groupby_menu.appendTo($buttons));
+            }
+            if (!this.options.disable_favorites) {
+                this.favorite_menu = new FavoriteMenu(this, this.query, this.dataset.model, this.action, this.favorite_filters);
+                menu_defs.push(this.favorite_menu.appendTo($buttons));
+            }
+        }
+        return menu_defs;
+    },
+    /**
+     * Will return $element where Filters, Group By and Favorite buttons going to push
+     * Will be overridden in mobile searchview so we can append buttons in specified element.
+     * @private
+     */
+    _getButtonsElement: function () {
```

Comment:
This method is not very useful.
It would be better to override this.$buttons in start.

Reply:
@adr-odoo While checking the reason, why we are not able to override this.$buttons, I found few things,
Searchview is not clean, it messy, somewhere we assign $searchview_buttons = searchview.$buttons and somewhere searchview.$buttons = $searchview_buttons

If we override this.$buttons then it is displayed in control panel which we don't want, in mobile view, we don't want to display buttons in control panel, it should be displayed in the separate screen.

We don't want to refactor searchview buttons so we pushed everything like Filter, Group By etc.. in different div than $buttons, maybe later when the whole searchview is refactored at that time we will improve it.